drivers/clk/Kconfig

```diff
@@ -163,6 +163,13 @@ config CLK_OCTEON
 	help
 	  Enable this to support the clocks on Octeon MIPS platforms.
 
+config CLK_RP1
```

Comment:
Probably the "drivers: clk: introduce clock driver for RP1"  should be marked as HACK, and commit msg should be definitely updated to say - it supports only NET clock.

Reply:
It is normal for U-Boot drivers to add only minimally needed set of features. Additional clocks can be added when they are required, driver design allows this with ease.

I can show example of exactly the same behavior in, say, QCOM drivers.

Reply:
This is a common practice in u-boot to add only clock which are in use right now. So I think this is definitely not a HACK. As for the NET clock - the clock name is RP1_CLK_ETH_TSU. I can add it to the commit description but wasn't able to find similar mentions in the u-boot history with the list of supported clocks. Also, please note that I have added all clock devices in rp1.h.